hack/ci/cloud-init/common.yaml.tpl

```diff
@@ -20,13 +20,13 @@ packages:
 package_upgrade: true
 write_files:
 - path: /etc/sysctl.d/devstack.conf
-  permissions: 0644
+  permissions: "0644"
```

Comment:
not sure whether it's because latest cloud-init has this kind of restriction? 
https://github.com/canonical/cloud-init/pull/1360/files#diff-c18cfd88ec0bd5f1077576fee00a0b2e87cf0b30aa547ad1eaacfd5742e4a757R1870